src/tutorial/testing.md

```diff
@@ -1,9 +1,9 @@
 # Testing
 
 Over the decades of people doing software development
-one truth has been found:
+a pattern has emerged:
```

Comment:
Nooo this takes all the fun out of it, let's keep this

Reply:
sure